.wiki/Configuration-Repository.md

```diff
@@ -391,3 +391,128 @@ When `storage` section under `meta` section in configured, it is also possible t
 using `docker push` command to proxy repository and store them locally. 
 
 Find the example how to pull and push images into docker registry in [Docker repository section](./Configuration-Repository.md#usage-example).
+
+## NPM
+
+NPM repository is the [repository for JavaScript](https://www.npmjs.com/) code sharing, packages 
+store and management. Here is the configuration example for NPM repository:
+
+```yaml
+repo:
+  type: npm
+  url: http://{host}:{port}/{repository-name}
+  storage:
+    type: fs
+    path: /var/artipie/data/
+  permissions:
+    "*":
+      - download
+```
+
+The NPM repository configuration requires `url` field that contains repository full URL, 
+`{host}` and `{port}` are Artipie service host and port, `{repository-name}`
+is the name of the repository (and repository name is the name of the repo config yaml file). Check 
+[storage](./Configuration-Storage.md) and [permission](./Configuration-Repository%20Permissions.md) 
+documentations to learn more about these settings.
+
+To use NPM repository with `npm` client, you can specify Artipie NPM repository with `--registry` option:
+```bash
+# to install the package
+npm install @hello/my-project-name --registry http://{host}:{port}/{repository-name}
+# to publish the package
+npm publish @hello/my-project-name --registry http://{host}:{port}/{repository-name}
+```
+or it's possible to set Artipie as a default registry:
+```bash
+npm set registry http://{host}:{port}/{repository-name}
+```
+
+## NPM Proxy
+
+NPM proxy repository will proxy all the requests to configured remote and store all the downloaded
+packages into configured storage:
+
+```yaml
+repo:
+  type: npm-proxy
+  path: {repository-name}
+  storage:
+    type: fs
+    path: /var/artipie/data/
+  settings:
+    remote:
+      url: http://npmjs-repo/
+```
+
+all the fields of YAML config are required, `path` is the repository relative path, [`storage` section](./Configuration-Storage.md)
```

Comment:
- all -> All
- it would be better to remove `` from the `storage` or exclude `storage` from the link